[grpc-health-check] Documentation & problem with published package

### Problem description
There is no documentation around how `grpc-health-check` library should be used.
I have tried to use it based on existing test cases, but there is major issue with latest published version `1.6.6`:
The "main" file defined in `package.json` file is `src/node/index.js` [[here](https://github.com/grpc/grpc-node/blob/990f082f0094f32cc9e1d8f272886955e98afc9c/packages/grpc-health-check/package.json#L30)] while there is no such file.
Hence, we can't have `require('grpc-health-check')` because it will throw `Cannot find module 'grpc-health-check'` error. 
It should be used like `require('grpc-health-check/health')`. 

But it can be resolved easily by fixing that line to point to `health.js` and publish it as `1.6.7` or something.

### Reproduction steps
```bash
mkdir /tmp/test && cd /tmp/test
npm init
npm i grpc-health-check@1.6.6
node -e "require('grpc-health-check')"
```
### Environment
 - Any node version / but i've tested on 8.8.1
clang 3.8.0-2ubuntu4]
 - grpc-health-check@1.6.6

### Additional context
I'd be happy to raise PRs to fix that and also add documentation if this is not deprecated or sth.
Also wondering why link to repository in npmjs.com is pointing to https://github.com/grpc/grpc/ 🤔

--- a/packages/grpc-health-check/README.md
+++ b/packages/grpc-health-check/README.md
@@ -1,0 +1,91 @@
+# grpc-health-check
+
+Health check client and service for use with gRPC-node.
+
+## Background
+
+This package exports both a client and server that adhere to the [gRPC Health Checking Protocol](https://github.com/grpc/grpc/blob/master/doc/health-checking.md).
+
+By using this package, clients and servers can rely on common proto and service definitions. This means:
+- Clients can use the generated stubs to health check _any_ server that adheres to the protocol.
+- Servers do not reimplement common logic for publishing health statuses.
+
+## Installation
+
+Use the package manager [npm](https://www.npmjs.com/get-npm) to install `grpc-health-check`.
+
+```bash
+npm install grpc-health-check
+```
+
+## Usage
+
+### Server
+
+Any gRPC-node server can use `grpc-health-check` to adhere to the gRPC Health Checking Protocol. 
+The following shows how this package can be added to a pre-existing gRPC server.
+
+```javascript 1.8
+// Import package
+let health = require('grpc-health-check');
+
+// Define service status map. Key is the service name, value is the corresponding status.
+// By convention, the empty string "" key represents that status of the entire server.
+const statusMap = {
+  "ServiceFoo": proto.grpc.health.v1.HealthCheckResponse.ServingStatus.SERVING,
+  "ServiceBar": proto.grpc.health.v1.HealthCheckResponse.ServingStatus.NOT_SERVING,
+  "": proto.grpc.health.v1.HealthCheckResponse.ServingStatus.NOT_SERVING,
+};
+
+// Construct the service implementation
+let healthImpl = new health.Implementation(statusMap);
+
+// Add the service and implementation to your pre-existing gRPC-node server
+server.addService(health.service, healthImpl);
+```
+
+Congrats! Your server now allows any client to run a health check against it.
+
+### Client
+
+Any gRPC-node client can use `grpc-health-check` to run health checks against other servers that follow the protocol.
+The following shows how this package can be used by a pre-existing gRPC client.
+
+```javascript 1.8
+// Import package
+let grpc = require('grpc');
+let health = require('grpc-health-check');
+
+// Create the client
+let healthClient = new health.Client('localhost:8082', grpc.credentials.createInsecure());
+
+// Define the request, which contains the service to health check
+// By convention, the empty string "" key represents that status of the entire server.
+let request = {
+  service: "",
+};
+
+// Define the callback
+function callback (error, status) {
+  if (error) {
+    console.error(error);
+    return;
+  }
+
+  console.log(status);
+}
+
+// Make the request
+healthClient.check(request, callback);
+```
+
+This should print out "NOT_SERVING" if used with the server above.
+
+## Contributing
+
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
+
+Please make sure to update tests as appropriate.
+
+## License
+[Apache License 2.0](https://choosealicense.com/licenses/apache-2.0/)

--- a/packages/grpc-health-check/README.md
+++ b/packages/grpc-health-check/README.md
@@ -49,37 +49,6 @@ Congrats! Your server now allows any client to run a health check against it.
 ### Client
 
 Any gRPC-node client can use `grpc-health-check` to run health checks against other servers that follow the protocol.
-The following shows how this package can be used by a pre-existing gRPC client.
-
-```javascript 1.8
-// Import package
-let grpc = require('grpc');
-let health = require('grpc-health-check');
-
-// Create the client
-let healthClient = new health.Client('localhost:8082', grpc.credentials.createInsecure());
-
-// Define the request, which contains the service to health check
-// By convention, the empty string "" key represents that status of the entire server.
-let request = {
-  service: "",
-};
-
-// Define the callback
-function callback (error, status) {
-  if (error) {
-    console.error(error);
-    return;
-  }
-
-  console.log(status);
-}
-
-// Make the request
-healthClient.check(request, callback);
-```
-
-This should print out "NOT_SERVING" if used with the server above.
 
 ## Contributing
 

--- a/packages/grpc-health-check/package.json
+++ b/packages/grpc-health-check/package.json
@@ -1,8 +1,8 @@
 {
   "name": "grpc-health-check",
-  "version": "1.7.0",
+  "version": "1.7.1",
   "author": "Google Inc.",
-  "description": "Health check service for use with gRPC",
+  "description": "Health check client and service for use with gRPC-node",
   "repository": {
     "type": "git",
     "url": "https://github.com/grpc/grpc-node.git"
@@ -22,6 +22,7 @@
   },
   "files": [
     "LICENSE",
+    "README.md",
     "health.js",
     "v1"
   ],

